services/eventservice/src/types.rs

```diff
@@ -0,0 +1,69 @@
+use chrono;
+use serde::{Deserialize, Serialize};
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub struct Category {
+    pub name: String,
+}
+
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub struct Event {
+    pub id: u64,
+    pub title: String,
+    pub cover: String,
+    pub description: String,
+    pub organizer: u64,
+    pub date_created: i64,
+    pub date_planning: i64,
+    pub category: Category,
+}
+
+impl Event {
+    pub fn new() -> Event {
+        Event {
+            id: 1,
+            title: "Unnamed".to_string(),
```

Comment:
Well "abc" is not String. In c++ "abc" is char* and in rust is &'static str.
As you know from memory layout, all static variables are stored in special place in program. So str stored there, it can't be modified and doesn't use any space in stack or heap
String is usually implemented like
```
struct String 
{
     size_t length;
     char * data;
}
```
So String's data is being stored in heap and can be modified. That's why we need convert str to String. Most languages do that, but usually implicitly, like c++